pygmt/helpers/decorators.py

```diff
@@ -15,13 +15,14 @@
 COMMON_OPTIONS = {
     "R": """\
```

Comment:
```suggestion
    "R": r"""\
```
Need to make this a raw r-string to avoid the pylint warning (W605 invalid escape sequence '\ ')

Reply:
It rendered incorrectly when I would have raw strings in both COMMON_OPTIONS and the function doc strings (check out the deployment for 62e8aad at the top).  My guess is some issue with a raw string with escape characters being passed into another raw string.

Reply:
I see, might need to rethink this a bit then or just ignore the `pylint` error if it's harmless.

Reply:
Should I add a `pylint: disable=invalid-escape-sequence` (is that the correct wording/syntax?)?